### DISCUSSION TITLE
Is it possible to configure partial through the go module?

### DISCUSSION THREAD

**Discussion by codeboten** [goreleaser/goreleaser#4033] [Q&A]
The configuration section shown here https://goreleaser.com/customization/partial/#customization describes a `partial` section, but I was unable to find any references to this in the go module (https://pkg.go.dev/github.com/goreleaser/goreleaser/pkg/config). Is there no way to specify this?

**Comment by caarlos0** [comment]
That option is only available in the Pro version, which is not open source.

**Reply by codeboten** [reply]
the opentelemetry-collector-releases repo uses the pro version, does this mean it's not possible to set that option through a go module or is there a different go module we should be using?

**Reply by caarlos0** [reply]
The pro version is only available from the binary releases.

You can check install methods [here](https://goreleaser.com/install/)

**Reply by codeboten** [reply]
Right, we use the github action for producing releases, but we use the library to generate the goreleaser configuration. You can see it here: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/cmd/goreleaser/internal/configure.go. I was hoping to be able to set the partial configuration via this library.

That is not currently supported correct?

**Reply by caarlos0** [reply]
ah yes, correct.

**Reply by codeboten** [reply]
Thanks for confirming. Just a quick follow up, is this something that you'd consider adding to the library if I opened an issue for it? If not we'll have to figure out what we're going to do w/ our currently generation tooling

**Reply by caarlos0** [reply]
hmmm I don't know, have to investigate if it wouldn't be confusing for other users...

PS: you could probably generate the structs from the yamlschema somehow?
